Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AnimViewportWidget.cpp

```diff
@@ -199,12 +197,11 @@ namespace EMStudio
     void AnimViewportWidget::UpdateCameraFollowUp(bool followUp)
     {
         const auto lookAndOrbitCameras =
-            AZStd::vector<AZStd::shared_ptr<AzFramework::CameraInput>>{ m_lookRotateCamera, m_lookTranslateCamera,
+            AZStd::vector<AZStd::shared_ptr<AzFramework::CameraInput>>{ m_lookTranslateCamera,
                                                                         m_lookScrollTranslationCamera, m_lookPanCamera, m_orbitCamera };
 
         const auto followCameras =
-            AZStd::vector<AZStd::shared_ptr<AzFramework::CameraInput>>{ m_followRotateCamera, m_followScrollDollyCamera,
-                                                                        m_followScrollMotionCamera };
+            AZStd::vector<AZStd::shared_ptr<AzFramework::CameraInput>>{ m_followRotateCamera, m_followScrollDollyCamera};
```

Comment:
You can remove this functionality if you want, but this is what happens in the main viewport when holding Alt and orbiting (RMB + Mouse Move up/down -> Dolly)

Reply:
I think the dolly behavior are fine but once it enters the follow camera option, the RMB + Mouse up/down become dolly WITHOUT needing the alt key.  So we have:
1) non follow up: RMB Mouse move up/down -> free look
2) follow up: RMB Mouse move u[/down -> dolly
That is super weird to me somehow.

Reply:
Okay cool understood, that change is fine to remove from me but I do think poking the DTs to have a play around with the controls would definitely be helpful too, that type of feedback can be invaluable